data/companies.json

```diff
@@ -496,6 +496,22 @@
         "youtube": "https://www.youtube.com/channel/UCNG0LqHlyecBNCcIAnAMtXw"
       }
     },
+    {
+      "name": "TechLekh Services Pvt. Ltd.",
+      "establishedAt": 1420099200,
+      "location": "Sukedhara, Kathmandu",
+      "website": "https://techlekh.com/",
+      "email": "contact@techlekh.com",
+      "phone": "+977-9813953967",
```

Comment:
Does this place have a landline number?
If yes, please provide the landline number.

Reply:
No, I didn't find one!